docs/source/get_started/installation.rst

```diff
@@ -57,8 +44,65 @@ using their OS package manager.
 .. _Python Downloads: http://python.org/download
 
 
+Enaml and Dependencies
+~~~~~~~~~~~~~~~~~~~~~~
+
+You can install enaml and (almost) all of its dependencies through the pip
+command line.::
+
+    $ pip install enaml
+
+Toolkit Rendering Library
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Enaml's declarative widgets provide a layer of abstraction on top of the
+widgets of a toolkit rendering library. You will need to install this
+dependency separately.
+
+Enaml supports using either PyQt4 or PyQt5 and uses the `QtPy`_ library
+as compatibility layer.
+
+The recommended library is `PyQt5`_,  a robust set of Python bindings to the
+Qt 5 toolkit.  (It includes the necessary parts of Qt 5.)
+
+On 32 and 64-bit Windows, 64-bit OS X and 64-bit Linux, with Python 
+versions >=3.5, it can be installed via pip::
+
+    $ pip install pyqt5
+
+An alternative is _PyQt_, which is a robust set of Python bindings to the 
+Qt 4 toolkit. It is no longer a supported project, but can still be installed
+and used with Enaml. The `PyQt Downloads`_ page contains Windows installers
+- which also the necessary parts of Qt 4. OSX users can install PyQt4 via
+`Homebrew`_. Linux users should install via the system package manager.
+
+Another alternative is `PySide`_ bindings to Qt. This is not recommended. The
+PySide bindings are not nearly as stable as PyQt. They contain several bugs
+which can and will cause applications to crash. There are also some API
+differences between the two libraries. So while some effort is made to
+support the use of PySide in Enaml, it is "use at your own risk".
+
+To activate PySide support, install PySide or PySide2 separately and set the
+environment variable ``QT_API=pyside`` (``QT_API=pyside2`` for PySide2)
+before starting the Enaml application.
+
+While Enaml is architected to be toolkit agnostic, the use of anything but
+`Qt`_-based toolkit libraries is not recommended.
```

Comment:
See @MatthieuDartiailh's other comment regarding different backends. I think this should be rewritten to say that Qt is the only officially supported backend.

Reply:
Fixed in next commit